cpp/src/parquet/encoding_test.cc

```diff
@@ -1413,5 +1436,16 @@ TYPED_TEST(TestDeltaBitPackEncoding, BasicRoundTrip) {
   }
 }
 
+TYPED_TEST(TestDeltaBitPackEncoding, ZeroRoundTrip) {
```

Comment:
Would this be a better name?

```suggestion
TYPED_TEST(TestDeltaBitPackEncoding, SmallBatchRoundTrip) {
```